src/index.ts

```diff
@@ -61,6 +61,11 @@ export type { ButtonProps } from './components/Button';
 export type { LinkProps } from './components/Link';
 export type { PageNotificationProps } from './components/PageNotification';
 export type { AppNotificationProps } from './components/AppNotification';
+export type {
+  DataTableProps,
+  DataTableRowProps,
+  DataTableWithStatus,
```

Comment:
Utility type for augmenting table model data with the details of the status type in the type object.